### PR TITLE
Try replaced with try-with-resources

### PR DESCRIPTION
**Pull request by olhabn**

### CODE REVIEW

File: src/foam/dao/HTTPSink.java [hunk 1]
```diff
@@ -64,12 +62,12 @@ public void put(Object obj, Detachable sub) {
       }
       conn.connect();
 
-      os = conn.getOutputStream();
-      writer = new BufferedWriter(new OutputStreamWriter(os, StandardCharsets.UTF_8));
-      writer.write(outputter.stringify((FObject)obj));
-      writer.flush();
-      writer.close();
-      os.close();
+      try(OutputStream os = conn.getOutputStream()) {
```
Comment (kgrgreer) [inline src/foam/dao/HTTPSink.java]: Add space after 'try', this line and next.

File: src/foam/nanos/geocode/GoogleMapsGeocodingDAO.java [hunk 2]
```diff
@@ -123,9 +122,10 @@ public void execute(X x) {
           conn.connect();
 
           builder.setLength(0);
-          reader = new BufferedReader(new InputStreamReader(conn.getInputStream()));
-          while ( (line = reader.readLine()) != null ) {
-            builder.append(line);
+          try(BufferedReader reader = new BufferedReader(new InputStreamReader(conn.getInputStream()))) {
```
Comment (kgrgreer) [inline src/foam/nanos/geocode/GoogleMapsGeocodingDAO.java]: add space after try

File: src/foam/dao/HTTPSink.java (hunk 1 above)
Comment (olhabn) [reply src/foam/dao/HTTPSink.java]: fixed

File: src/foam/nanos/geocode/GoogleMapsGeocodingDAO.java (hunk 2 above)
Comment (olhabn) [reply src/foam/nanos/geocode/GoogleMapsGeocodingDAO.java]: fixed

**Pull request merged**